octopuslite/__init__.py

```diff
@@ -1 +1,2 @@
 from .reader import DaskOctopusLiteLoader
+from .utils import image_generator
```

Comment:
Do you still want this generator?

Reply:
up to you, i only use it once in the pipeline for the alignment step so we could remove it and i could just have it as a function in that notebook

Reply:
i will remove